content/en/tracing/guide/monitor-kafka-queues.md

```diff
@@ -0,0 +1,105 @@
+
+### Integration
+
+Easily visualize the performance of your cluster in real time qnd correlate the performance of Kafka with the rest of your applications by using [Datadog's Kafka integration][1]
+
+{{< img src="tracing/guide/monitor_kafka_queues/kafka_dashboard.png" alt="Data Streams Monitoring Demo">}}
+<br>
+
+### Data Stream Monitoring
+
+[Datadog Data Streams Monitoring][2] provides a standardized method for your teams to measure pipeline health and end-to-end latencies for events traversing across your system. The deep visibility offered by Data Streams Monitoring enables you to pinpoint faulty producers, consumers, or queues driving delays and lag in the pipeline; discover hard-to-debug pipeline issues such as blocked messages, hot partitions, or offline consumers; and collaborate seamlessly across relevant infrastructure or app teams.
+
+{{< img src="tracing/guide/monitor_kafka_queues/dash-2022-data-streams-compressed-blurb.mp4" alt="Data Streams Monitoring Demo" video="true">}}
+<br>
+
+### Collect distributed traces
+
+Datadog APM and distributed tracing gives you expanded visibility into the performance of your services by measuring request volume and latency. You create graphs and alerts to monitor your APM data, and you can visualize the activity of a single request in a flame graph like the one shown below to better understand the sources of latency and errors.
+<br>
+{{< img src="tracing/guide/monitor_kafka_queues//kafka_trace.png" alt="A kafka consumer span" >}}
+<br>
+Datadog APM can automatically trace requests to and from Kafka clients. This means you can collect traces without modifying your source code. Note that we inject headers in the kafka messages to be able to propagate the context of the trace from the producer to the consumer.
+
+#### Setup
+
+To trace kafka applications, we actually trace the producing and consuming calls within the Kafka SDK. So to monitor Kafka, you just have to setup APM on your services. See [the documentation][3] for guidance on getting started with APM and distributed tracing.
+
+#### Monitor in the APM UI
+
+A classic Kafka setup will show a trace with a producer span, and as a child, a consumer span. Any work that would generate a trace in the consumption side will be represented by child spans of the consumer span.
+Each spans contain a set of tags with the `messaging` prefix. Below, the tags you can find on Kafka spans.
+
+<div class="alert alert-info">
+  <div class="alert-info">
+    <div>To get a more global view of Datadog's spans metada, please refer to the <a href="/tracing/trace_collection/tracing_naming_convention">Span Tags Semantics documentation</a></strong>.</div>
+  </div>
+</div>
+
+| Tag                             | Description                                                                                        | Availability         |
```

Comment:
It may be better to stay consistent with the description's in https://docs.datadoghq.com/tracing/trace_collection/tracing_naming_convention/#message-queue ? Currently, both docs share the same tags but the descriptions are different. What do you think ?

Reply:
Indeed, i didn't know this page existed when I copy pasted that from Confluence :/ Fixed in d674520d612c9a53638d16682090771de6c96843